lib/inferno/apps/cli/services/docker-compose.global.yml

```diff
@@ -4,7 +4,7 @@ services:
     image: infernocommunity/fhir-validator-service
     # Update this path to match your directory structure
     volumes:
-      - ./data/igs:/home/igs
+      - ~/.inferno/data/igs:/home/igs
```

Comment:
We can't just change this directory in the docker compose because in multiple places inferno core writes to `data/igs`, not `~/.inferno/data/igs`.